types/index.ts

```diff
@@ -756,36 +756,36 @@ declare module 'mathjs' {
   expectTypeOf(math.testFun()).toMatchTypeOf<number>()
 
   expectTypeOf(math.import({
-    myvalue: 42,
-    simplify: (name: string) => `simplify ${name}`
+    myvalue1: 42,
+    myFunc: (name: string) => `myFunc ${name}`
   })).toMatchTypeOf<void>()
 
   expectTypeOf(math.import({
-    myvalue: 42
+    myvalue2: 42
   }, {
     override: true
   })).toMatchTypeOf<void>()
 
```

Comment:
Why was it necessary to change to `myvalue2` here? Doesn't the `override` option make it OK to redefine an existing symbol?

Reply:
Oh, no that probably wasn't necessary. Want me to change?

Reply:
I mean, leaving it as myvalue when all the other tests use a new name does sort of highlight/test the meaning of override...

Reply:
True. Will fix